packages/account/src/Components/poi/status/unsupported/card-details/documents-upload.tsx

```diff
@@ -120,20 +121,22 @@ const DocumentsUpload = ({ initial_values, is_from_external, data, goToCards, on
                                 </div>
                             </div>
                             <div className={`${ROOT_CLASS}__btns`}>
```

Comment:
I think u can remove the `div` and use the div's class in `formfooter`

Reply:
sure, 👍 
will do that